io_scene_niftools/modules/nif_import/object/__init__.py

```diff
@@ -146,7 +149,7 @@ def import_object_flags(n_block, b_obj):
 
         if hasattr(n_block, "data") and isinstance(n_block.data.consistency_flags, NifClasses.ConsistencyType):
             b_obj.niftools.consistency_flags = n_block.data.consistency_flags.name
-        if n_block.is_skin():
+        if n_block.is_skin() and hasattr(n_block, "skin_instance"):
```

Comment:
Should this perhaps go into is_skin()?

Reply:
For BSTriShapes and NiTriShapes, it already is. However, NiMeshes can also be skins (at least how I've put it in new-pyffi), and they don't have a skin_instance.